en/docs/integrations/integration-concepts.md

```diff
@@ -1,57 +1,54 @@
 # Integration Concepts
-Before you start building integrations with Choreo, there are basic concepts that you need to understand. This section walks you through all the integration concepts you need to know. 
- 
+Choreo integrations allow you to create enterprise integrations to automate workflows in a user-friendly low-code environment. Here are some basic concepts which will help you understand Choreo integrations as you develop your integrations in the [Choreo Console]({{choreo_console}}).
+  
 ## Integration
-Integration in Choreo is the process of connecting APIs, events, and streams to function as a single unit that can share information.
-
-## Integration Template
-An integration template in Choreo is a prebuilt sample designed to address a common integration requirement. Choreo provides a comprehensive set of integration templates for you to select and use depending on your requirement.
+Integration in Choreo is a process that connects APIs, events, and streams to function as a single unit. Choreo provides an integration template which is a pre-built sample designed to address a common integration requirement. Choreo includes a set of comprehensive integration templates for you to select and use.
 
 ## Trigger
-A trigger in Choreo integrations is an event or an action that can cause an integration to start executing.
+A trigger in Choreo integrations is an event or an action that can cause the integration to start executing.
 
 Choreo supports the following trigger types:
 
- - **Schedule**: To trigger an integration based on a schedule that you set.
- - **Manual**: To run and test an integration manually.
- - **Calendar**: To trigger an integration based on events in a Google Calendar.
- - **GitHub**: To trigger an integration based on GitHub events and actions.
- - **Salesforce**: To trigger an integration based on topics in a Salesforce account. 
+ - **Schedule**: Triggers execution according to a given schedule.
+ - **Manual**: Triggers execution manually by clicking 'Run & Test'.
```

Comment:
This definition is incorrect. `Run & Test` is valid for all the trigger types. The difference is not on how we run it, but how we designs in. Manual is free form.

Reply:
Fixed.